lyber-core.gemspec

```diff
@@ -1,34 +1,35 @@
-# -*- encoding: utf-8 -*-
-lib = File.expand_path('../lib/', __FILE__)
+lib = File.expand_path('lib', __dir__)
 $:.unshift lib unless $:.include?(lib)
 
 Gem::Specification.new do |s|
-  s.name        = "lyber-core"
-  s.version     = "4.1.3"
+  s.name        = 'lyber-core'
+  s.version     = '4.1.3'
   s.licenses    = ['Apache-2.0']
   s.platform    = Gem::Platform::RUBY
-  s.authors     = ["Alpana Pande","Bess Sadler","Chris Fitzpatrick","Douglas Kim","Richard Anderson","Willy Mene","Michael Klein", "Darren Weber","Peter Mangiafico"]
-  s.email       = ["sul-devops-team@lists.stanford.edu"]
-  s.homepage    = "http://github.com/sul-dlss/lyber-core"
-  s.summary     = "Core services used by the SUL Digital Library"
-  s.description = "Contains classes to make http connections with a client-cert, use Jhove, and call Suri\n" +
-                    "Also contains core classes to build robots"
+  s.authors     = ['Alpana Pande', 'Bess Sadler', 'Chris Fitzpatrick', 'Douglas Kim', 'Richard Anderson', 'Willy Mene', 'Michael Klein', 'Darren Weber', 'Peter Mangiafico']
```

Comment:
hahahaha - only @peetucket still works here ...

Reply:
Literally haven't even heard of a couple of these people.